site/kubernetes.md

```diff
@@ -0,0 +1,90 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator).
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -domain_resource_file /tmp/operator/domain-resource.yaml
+```
+
+For the simplest case, the Extract Domain Resource Tool will create a sparse domain file. This is what is generated when there is not `kubernetes` section in the model, or that section is empty.
+```yaml
+apiVersion: weblogic.oracle/v6
+kind: Domain
+metadata:
+    name: DemoDomain
+spec:
+    domainHome: /u01/mydomain
+    image: '--FIX ME--'
+    imagePullSecrets:
+    -   name: '--FIX ME--'
+    webLogicCredentialsSecret: '--FIX ME--'
+    clusters:
+    -   clusterName: mycluster
+        replicas: 2
+    -   clusterName: mycluster3
+        replicas: 4
+```
+
+For this example, the user is expected to fill in the image and secrets information identified by `--FIX ME--` in the domain resource output. The value for `domainHome` was set from the command line. The `kind` and `name` were set to the domain name derived from the topology section of the model, or the default `base_domain`. The cluster entries are pulled from the topology section of the model, and their replica counts were derived from the number of servers for each cluster.
```

Comment:
I think that we should guide users to replace --FIX ME-- as ' @@PROP:param@@' in a properties file.

Reply:
Agreed, I'll add it to the sample model with some details